functional/MBCS_Tests/codepoint/build.xml

```diff
@@ -27,23 +27,16 @@ limitations under the License.
 	<property name="src" location="./src" />
 	<property name="build" location="./bin" />
 	
-        <condition property="not_se80" value="true">
-           <and>
-             <not>
-               <equals arg1="${JAVA_VERSION}" arg2="SE80" />
-             </not>
-             <not>
-               <equals arg1="${JDK_VERSION}" arg2="8" />
-             </not>
-           </and>
-        </condition>
+	<condition property="not_se80" value="false" else="true">
+			<equals arg1="${JDK_VERSION}" arg2="8" />
+	</condition>
```

Comment:
Personally I feel this is quite confusing using double negtive. I would prefer 
	
        <condition property="not_se80" value=""true">
		<not>	
                       <equals arg1="${JDK_VERSION}" arg2="8" />
               </not>
	</condition>
Probably could also rename property as '8above'?